alonzo/impl/src/Cardano/Ledger/Alonzo.hs

```diff
@@ -58,3 +71,10 @@ instance
 instance CC.Crypto c => ValidateAuxiliaryData (AlonzoEra c) where
   hashAuxiliaryData x = AuxiliaryDataHash (hashAnnotated x)
   validateAuxiliaryData = error ("NO validateAuxiliaryData yet.") -- TODO Fill this in
+
+proxy :: Proxy (AlonzoEra c)
+proxy = Proxy
+
+type Era c = AlonzoEra c
```

Comment:
This is a very confusing export. We use `Era` all over the place, it always refers to the Era class. I think it's a bad idea to introduce this alias.